staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/oidc.go

```diff
@@ -298,6 +300,15 @@ func (a *OIDCAuthenticator) parseTokenClaims(claims jose.Claims) (user.Info, boo
 			info.Groups[i] = a.groupsPrefix + group
 		}
 	}
-
 	return info, true, nil
 }
+
+type claims map[string]json.RawMessage
+
+func (c claims) unmarshalClaim(name string, v interface{}) error {
+	val, ok := c[name]
+	if !ok {
+		return fmt.Errorf("claim not present")
+	}
+	return json.Unmarshal([]byte(val), v)
```

Comment:
does this echo the value in errors? we weren't returning claim values in error messages before, just type info

Reply:
It's doesn't appear to return the value: https://play.golang.org/p/Ed2vi-1gDEK

I can add a test to make sure that behavior doesn't change in future releases of Go.